### PR TITLE
Release - migration to routify

### PR DESCRIPTION
**Pull request by Schparky**
### Changed
- use routify router (was svelte-spa-router)
### Fixed
- Freshdesk widget wasn't loading until page refresh

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -1,76 +1,75 @@
-import svelte from 'rollup-plugin-svelte'
-import resolve from 'rollup-plugin-node-resolve'
-import commonjs from 'rollup-plugin-commonjs'
```
Comment (hobbitronics) [inline rollup.config.js:3]: Just curious why the import changed.

File: src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<script>
+import Request from '../../../../../components/Request.svelte'
```
Comment (hobbitronics) [inline src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte:2]: indentation

**Review by hobbitronics — APPROVED** [review]

File: rollup.config.js (hunk 1 above)
Comment (Schparky) [reply rollup.config.js:3]: It was moved: https://github.com/rollup/rollup-plugin-commonjs#moved

File: src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte (hunk 2 above)
Comment (Schparky) [reply src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte:2]: There's an unfortunate mix of indentation of <script> code throughout. Probably worth a separate PR just for whitespace cleanup.

**Review by kode-monki — APPROVED** [review]

**Pull request merged**